backend/lucene/src/main/java/org/hibernate/search/backend/lucene/types/dsl/impl/LuceneStringIndexFieldTypeOptionsStep.java

```diff
@@ -164,13 +165,15 @@ public LuceneIndexValueFieldType<String> toIndexFieldType() {
 			}
 		}
 
+		FieldType fieldType = getFieldType( resolvedProjectable, resolvedSearchable, analyzer != null, resolvedNorms, resolvedTermVector );
 		LuceneStringFieldCodec codec = new LuceneStringFieldCodec(
-				getFieldType( resolvedProjectable, resolvedSearchable, analyzer != null, resolvedNorms, resolvedTermVector ),
+				fieldType,
 				docValues,
 				indexNullAsValue,
 				builder.indexingAnalyzerOrNormalizer()
 		);
 		builder.codec( codec );
+		builder.storeTermVectorOffsets( fieldType.storeTermVectorOffsets() );
```

Comment:
```suggestion
		builder.storeTermVectorOffsets( fieldType!=null && fieldType.storeTermVectorOffsets() );
```
would help with the failing build, but as we are going to introduce `Highlightable` I wonder if it wouldn't be better to just drop this last commit for the message and do it as part of the `Highlightable` PR ?

Reply:
Well if it's there and it passes with just this change, you might as well keep it.